openmrs-19/src/main/java/org/motechproject/openmrs19/domain/Encounter.java

```diff
@@ -18,6 +18,7 @@
     private Date encounterDatetime;
     private Patient patient;
     private Person provider;
+    private List<Person> encounterProviders;
```

Comment:
OpenMRS 1.12 returns json with list named encounterProviders. This name must be 'encounterProviders'